src/BayesFilters/src/EstimatesExtraction.cpp

```diff
@@ -117,11 +117,11 @@ std::vector<std::string> EstimatesExtraction::getInfo() const
 }
 
 
-VectorXf EstimatesExtraction::mean(const Ref<const MatrixXf>& particles, const Ref<const VectorXf>& weights) const
+VectorXd EstimatesExtraction::mean(const Ref<const MatrixXd>& particles, const Ref<const VectorXd>& weights) const
 {
-    VectorXf out_particle = VectorXf::Zero(7);
-    float    s_ang        = 0;
-    float    c_ang        = 0;
+    VectorXd out_particle = VectorXd::Zero(7);
+    double    s_ang        = 0;
+    double    c_ang        = 0;
```

Comment:
Here the indentation is wrong. There is an extra space before `s_ang` and `c_ang`.